src/mscorlib/src/System/Environment.cs

```diff
@@ -1519,8 +1583,11 @@ public static string UserDomainName
         {
             [System.Security.SecuritySafeCritical]  // auto-generated
             get {
+#if PLATFORM_UNIX
+                return MachineName;
```

Comment:
I see this is what Mono does. Perhaps we should, then.

---
In reply to: [68615139](https://github.com/dotnet/coreclr/pull/6006#discussion_r68615139) [](ancestors = 68615139)